Multisig/Features/Security/SensitiveStore.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  SensitiveStore.swift
+//  Multisig
+//
+//  Created by Dirk Jäckel on 30.11.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class SensitiveStore: EncryptedStore {
+    var initialized = false
+    private let store: KeychainItemStore
+    static let sensitivePublicKeyTag = "global.safe.sensitivePublicKeyTag"
+    static let sensitivePublicKEKTag = "global.safe.sensitivePublicKEKTag"
+    static let sensitivePrivateKEKTag = "global.safe.sensitivePrivateKEKTag"
+    static let sensitiveEncryptedPrivateKeyTag = "global.safe.sensitive.private.key.as.encrypted.data"
+    static let derivedPasswordTag = "global.safe.password.as.data"
```

Comment:
should all these tags be also private?

Reply:
They are used when creating  KeychainItems.

Reply:
ok, but are these constants are only used within this class?